style.css

```diff
@@ -297,14 +297,108 @@ button:active {
   justify-content: space-around;
 }
 
-.card-tags > ul > li {
+.card-work > .card-tags > li {
   background: lightgray;
   font-size: 12px;
   padding: 0.5em;
   margin: 0 0.5em;
   border-radius: 2px;
 }
```

Comment:
- [ ] Please make sure that the popup window background matches the one in the Figma design 

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/74744787/161864631-6b7429bc-9d8c-4001-abaa-4cb8cc40cadb.png) |  ![image](https://user-images.githubusercontent.com/74744787/161865126-b34ce705-3c85-4963-8efd-6966d25e3db3.png) |

Reply:
- [ ] Kindly work on the popup modal to match the one given in the Figma design just as the previous reviewer requested.

**Hints**
- Work on the width of the modal as it's stretching 100% of the browser window
- Also, please work on the close button. Currently, the close button is not doing what it is supposed to do - close the modal window.
- Kindly make the close button work so when it's clicked, the modal should be closed. 

**This is how your design should look like when the popup shows**

![image](https://user-images.githubusercontent.com/50941074/162048339-c79a0050-6e2b-44ae-898c-ea853be8f897.png)

**Your design**
![image](https://user-images.githubusercontent.com/50941074/162048823-c9e0e4ea-a4df-4fa4-a3ad-a4067175ef25.png)